### ISSUE TITLE
Empty new notes on Firefox

### ISSUE THREAD

**Issue by zeus-web** [davecburke/silverbullet-clipper#11]
Hi,

I just realized, that while on Chrome version everything is fine, the plugin does not work for me on Firefox:

New notes are created within the correct Folder and with the correct title. 
However, the new Note does not contain any Text (neither Body nor any frontmatter).

Just for reference, my environment:
- Firefox  115.15.0
- running under Debian Linux
- Silverbullet running locally via Docker

Thanks a lot for the very useful clipper and the perfect support!

**Comment by davecburke** [comment]
I've just downloaded Firefox 115.15.0. I'm running SilverBullet on docker locally using Ubuntu and I don't have an issue with or without a token. For me it operates the same as the chrome version 1.2.2. Are you able to send me the link that you're trying to clip? Thanks

**Comment by zeus-web** [comment]
Aargh... completely my fault! So sorry...

I entered the Host-Url with a trailing "/" by mistake ("http://10.1.1.127:3000/" instead of "http://10.1.1.127:3000").
That caused the issue!

After correction everything works as expected!

**Sorry again** for generating useless work and confusion ... ;-)

(Additionally I found out, that in fact the clipper notes were **not generated at all** - but after clicking the "go to quick note" link, which leads to a non-existing note, silverbullet created it as empty...)

**Comment by zeus-web** [comment]
Aargh... completely my fault! So sorry...

I entered the Host-Url with a trailing "/" by mistake ("http://10.1.1.127:3000/" instead of "http://10.1.1.127:3000").
That caused the issue!

After correction everything works as expected!

**Sorry again** for generating useless work and confusion ... ;-)

(Additionally I found out, that in fact the clipper notes were **not generated at all** - but after clicking the "go to quick note" link, which leads to a non-existing note, silverbullet created it as empty...)
